CMake/RealmCore.cmake

```diff
@@ -179,7 +180,11 @@ function(download_realm_core core_version)
         download_realm_tarball(${url_release} ${core_directory_release} ${core_library_release})
     else()
         if(APPLE)
-            set(basename "realm-core-cocoa")
+            if(REALM_CORE_PACKAGING STREQUAL "1")
```

Comment:
I don't feel strongly about it either way. You've done the flag, so we may as well stick with it until we drop support for the old packaging.